### PR TITLE
WIP compatibility with workarea 3.4

### PR DESCRIPTION
**Pull request by roseliux**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -12,6 +12,6 @@ gemspec
 # your gem to rubygems.org.
 
 # To use a debugger
-# gem 'byebug', group: [:development, :test]
+gem 'pry-rails', group: [:development, :test]
```
Comment (bencrouse) [inline Gemfile:15]: Please revert debugging/unrelated changes before pushing. Thanks!

Comment (roseliux) [reply Gemfile:15]: understand, I was just debugging 😄

**Comment by roseliux** [comment]
@bencrouse push some changes but still not sure how the else clause should look, can you give me a hint. 
Thanks 😄

**Comment by roseliux** [comment]
I was confused about putting some config on the else clause but I figure you need to configure that when you install this plugin on your e-commerce, let me know if this PR needs more updates on the README or anything.
Thanks 😄 

@bencrouse  @jyucis

**Comment by roseliux** [comment]
thanks @bencrouse  I pushed default configuration on the else clause

File: workarea-orderbot.gemspec [hunk 2]
```diff
@@ -16,5 +16,5 @@ Gem::Specification.new do |spec|
 
   spec.files = `git ls-files`.split("\n")
 
-  spec.add_dependency 'workarea', '~> 3.5.x'
+  spec.add_dependency 'workarea', '~> 3.4.x'
```
Comment (bencrouse) [inline workarea-orderbot.gemspec:19]:
```suggestion
  spec.add_dependency 'workarea', '>= 3.4.x'
```

**Comment by bencrouse** [comment]
@roseliux this was released in v1.0.1, which is up on rubygems.org. Thanks for contributing.
